features/step_definitions/style_guide_steps.rb

```diff
@@ -0,0 +1,8 @@
+When(/^I visit the style guide$/) do
+  visit '/style-guide'
+end
+
+Then(/^the page is rendered correctly$/) do
+  # check that header from layout has rendered
+  expect(page).to have_css('header.navbar')
```

Comment:
@benlovell the best option I found was to test that the layout was correctly rendered.

The other option I considered as a possibility was a controller unit test, but this felt like it was putting a view test in the controller spec.

Any thoughts on a better testing process welcome.